components/TrialForm.tsx

```diff
@@ -0,0 +1,353 @@
+
+    setSubdomain(extractedDomain);
+  };
+
+  const onEditSubdomainClick = (e) => {
+    setEditSubdomain(!editSubdomain);
+  };
+
+  const errorHandler = new ErrorHandler();
+
+  errorHandler.subscribe("result", (e) => {
+    const error: FormError = { email: null, generic: null, subdomain: null };
+    const message = e?.message || e.toString();
+
+    if (message.includes("subdomain")) {
+      error.subdomain = "Subdomain already in use.";
+    } else if (message === "email must be unique") {
+      error.email = "Email address already in use.";
+    } else if (message === "Validation isEmail on email failed") {
+      error.email = "Please enter a valid email address.";
+    } else {
+      error.generic = message;
+    }
+    setError(error);
+  });
+
+  const { execApi } = useApi(errorHandler);
+
+  const onSubmit = async (data) => {
+    setLoading(true);
+    // defaults
+    let source = "/trial";
+    let medium = "web";
+    let campaign = globalThis?.document?.referrer;
+
+    const storage = globalThis?.sessionStorage;
+    if (storage) {
+      source = storage.getItem("utm_source")
+        ? storage.getItem("utm_source")
+        : storage.getItem("prevPath")
+        ? storage.getItem("prevPath")
+        : source;
+      medium = storage.getItem("utm_medium")
+        ? storage.getItem("utm_medium")
+        : medium;
+      campaign = storage.getItem("utm_campaign")
+        ? storage.getItem("utm_campaign")
+        : campaign;
+    }
+
+    data.subdomain = subdomain;
+    data.source = source;
+    data.medium = medium;
+    data.campaign = campaign;
+    data.email = data.email.toLowerCase();
+    data.companyWebsite = data.companyWebsite.toLowerCase();
+    data.subdomain = data.subdomain.toLowerCase();
+
+    const response = await execApi("post", `/api/v1/trial-request`, data);
+    if (response?.trialRequest) {
+      setRegistered(true);
+
+      if (isBrowser() && globalThis?.gtag) {
+        globalThis.gtag("event", "conversion", {
+          send_to: GoogleAdsTrialConversion,
+          event_callback: () => {},
+        });
+        globalThis.gtag("event", "trial-request", {
+          event_category: "conversion",
+          event_callback: () => {},
+        });
+      }
+    }
+    setLoading(false);
+  };
+
+  const Error = function ({ message }) {
+    if (!message) {
+      return null;
+    }
+    return <div style={{ fontSize: "smaller", color: "red" }}>{message}</div>;
+  };
+
+  const privacyPolicyLabel = (
+    <>
+      I agree to the{" "}
+      <Link href="/legal/privacy" passHref>
+        <a target="_blank" rel="noopener noreferrer">
+          privacy policy
+        </a>
+      </Link>
+    </>
+  );
+  const disabled = loading || registered;
+
+  return (
+    <>
+      <Error message={error.generic} />
+      {!registered ? (
+        <Container className="bg-white border shadow border-3 rounded col-11 p-4 mt-4 mt-md-0 me-4 me-lg-0">
+          <h3 className="mx-auto text-center">
```

Comment:
All of the field labels being centered looks odd to me.
![Screen Shot 2022-01-06 at 2 31 54 PM](https://user-images.githubusercontent.com/633226/148461678-838013f9-bf2a-4a23-a5a1-8a7abd3fc09c.png)

Can "First Name", "Last Name", "Work Email", and "Company Website" be left-aligned?

Reply:
<img width="1470" alt="Screen Shot 2022-01-06 at 3 01 13 PM" src="https://user-images.githubusercontent.com/303226/148464525-e0c376d6-cc95-4b8d-9690-e758d0d52aaf.png">
I'm not feeling left-align personally..., especially with other parts of the box being center aligned

Reply:
What if we left-align everything?

Reply:
I think that woudn't look great either - the box to me looks like it's 'hovering' above with greater Z-depth, so it sort of wants to be it's own different centered thing.  That's the vibe how it's used on the /trial page too

Reply:
I feel strongly that the labels should be left-aligned and everything else is centered. It matches what we already do on `/trial`.

Reply:
Then you've got it! left-aligned via https://github.com/grouparoo/www.grouparoo.com/pull/866/commits/870e4474babddbb8e390326de8354c31fe9fd7eb